underwater camera simulation

gustavoneves12 opened the pull request:
create Rock Component to set up an ocean environment for visualization
grab rendered image and sends to output port
set the camera position
create script to test the Rock Component
change the package name

doudou commented:
Will merge after the name question is resolved (since this commit contains the renaming changes)

gustavoneves12 committed:
underwater camera simulation
create Rock Component to set up an ocean environment for visualization
grab rendered image and sends to output port
set the camera position
create script to test the Rock Component
rename the package name

gustavoneves12 commented:
@doudou: PING
